src/web/app/pages-instructor/instructor-home-page/instructor-home-page.component.html

```diff
@@ -8,7 +8,7 @@ <h1>Home</h1>
 </div>
 
 <div class="row mb-3" *tmIsLoading="!hasCoursesLoaded">
-  <div class="col-4">
+  <div class="col mt-auto">
```

Comment:
One last bit, let's bring back `col-4` and not remove it though it does not have any effect but the container holding the other buttons has a width contained with `col-8`.
```suggestion
  <div class="col-4 mt-auto">
```

Update snapshot too 👍🏻